pubsub.go

```diff
@@ -0,0 +1,125 @@
+package lazyhttp
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/dendhi31/lazyhttp/redismaint"
+)
+
+func (httprequest *Client) Consumer() {
```

Comment:
I think we should return something on this function, the behaviour should be similar like http.ListenAndServe